### PR TITLE
1287 cards refactor

### PR DESCRIPTION
**Pull request by RomE-HK**
Task https://github.com/jdi-testing/jdi-light/issues/1287

### CODE REVIEW

**Comment by codacy-review** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- jdi-light-bootstrap/src/main/java/com/epam/jdi/light/ui/bootstrap/asserts/CardAssert.java  1
- jdi-light-bootstrap-tests/src/main/java/io/github/com/sections/card/CardImageCaps.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/jdi-light/jdi-light/pullRequest?prid=4515172&bid=15212842)

**Comment by RomE-HK** [comment]
All tests runs as before refactoring

**Comment by codacy-review** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/card/CardImageOverlaysTests.java  1
- jdi-light-bootstrap/src/main/java/com/epam/jdi/light/ui/bootstrap/asserts/CardAssert.java  1
- jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/card/CardImageCapsTests.java  1
- jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/card/CardImageTests.java  1
- jdi-light-bootstrap-tests/src/main/java/io/github/com/sections/card/CardImageCaps.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/jdi-light/jdi-light/pullRequest?prid=4515172&bid=15212842)

File: jdi-light-bootstrap-tests/src/main/java/io/github/com/sections/card/CardHorizontal.java [hunk 1]
```diff
@@ -1,13 +1,17 @@
...
+public class CardHorizontal extends Card {
+    @UI(".card-title")
+    public Text title;
+    @UI("//p[contains(text(), 'fictional character')]")
+    public Text mainText;
+    @UI(".text-muted")
+    public Text smallText;
+    @UI(".card-img")
+    public Image image;
 }
```
Comment (mf-code) [inline jdi-light-bootstrap-tests/src/main/java/io/github/com/sections/card/CardHorizontal.java]: please add new line

**Comment by mf-code** [comment]
please resolve conflicts

File: jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/card/CardGroupsTests.java [hunk 2]
```diff
@@ -31,70 +31,70 @@ public void before() {
...
+        cardGroupSectionWithFooter.card1.image.is().src(card1ImageSrc);
+        cardGroupSectionWithFooter.card2.image.is().src(card2ImageSrc);
+        cardGroupSectionWithFooter.card1.image.is().alt(card1ImageAlt);
+        cardGroupSectionWithFooter.card2.image.is().alt(card2ImageAlt);
+        cardGroupSectionWithFooter.card1.mainText.is().text(card1MainText);
+        cardGroupSectionWithFooter.card2.mainText.is().text(card2MainText);
+        cardGroupSectionWithFooter.card1.footerText.is().text(card1HeroName);
+        cardGroupSectionWithFooter.card2.footerText.is().text(card2HeroName);
     }
 }
```
[the remark this answers is not in the document]
Comment (RomE-HK) [reply jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/card/CardGroupsTests.java]: Fixed

File: jdi-light-bootstrap-tests/src/main/java/io/github/com/sections/card/CardHorizontal.java (hunk 1 above)
Comment (RomE-HK) [reply jdi-light-bootstrap-tests/src/main/java/io/github/com/sections/card/CardHorizontal.java]: Fixed

**Comment by codacy-review** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- jdi-light-bootstrap-tests/src/main/java/io/github/com/sections/card/CardImageCaps.java  1
- jdi-light-bootstrap/src/main/java/com/epam/jdi/light/ui/bootstrap/asserts/CardAssert.java  1
- jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/card/CardImageCapsTests.java  1
- jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/card/CardImageOverlaysTests.java  1
- jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/card/CardImageTests.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/jdi-light/jdi-light/pullRequest?prid=4515172&bid=15212842)

**Pull request merged**